### PR TITLE
DS-977

### PR DESCRIPTION
**Pull request by sugi-chan**
Added docs folder containing code to generate docs for readthedocs.io [here](https://creevey-sugi.readthedocs.io/en/latest/) is a sample that I put up to test that it would compile successfully on readthedocs (will get rid of it once we finish the pr). I used apparate as a basis as suggested.

-changed the changelogs and versions from 0.3.1 -> 0.3.2 
-new foder 'docs' includes files to generate the hosted docs
-Makefile and make.bat were generated as was conf.py. I put you as the author in line 24 of the conf.py file. 
-contrib.rst is based on the conduct.md and contributing.md files
-index.rst contains the general creevey info

### CODE REVIEW

**Comment by gsganden** [comment]
Please PR into WIP

**Comment by gsganden** [comment]
I think the version should go to 0.4.0 because you're adding a new feature.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+# [0.3.2] - 2019-02-13
```
Comment (gsganden) [inline CHANGELOG.md]: Should be 0.4.0, I think, because you are adding a feature.

File: creevey/_version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '0.3.1'
+__version__ = '0.3.2'
```
Comment (gsganden) [inline creevey/_version.py]: 0.4.0

**Comment by gsganden** [comment]
How do the generated files get generated? Do they need to be tracked by git?

**Comment by gsganden** [comment]
> contrib.rst is based on the conduct.md and contributing.md files

Should we delete the .md files?

File: CHANGELOG.md (hunk 1 above)
Comment (sugi-chan) [reply CHANGELOG.md]: adjusted.

File: creevey/_version.py (hunk 2 above)
Comment (sugi-chan) [reply creevey/_version.py]: adjusted.

**Comment by sugi-chan** [comment]
Files were almost all generated by sphinx as defaults (Makefile and make.bat are default generated). conf.py, contrib.rst, and index.rst are the only ones that should get changed since there is creevey info in those. I guess we can track them?

As far as deleting the md files, apparate has the md files  in the gitrepo and the info in the hosted docs so I think it is alright to leave them. 

I'll go open a new PR into WIP

**Pull request closed without merging**